### ISSUE TITLE
Pasting url from Google Maps crashes editor in Chrome

### ISSUE THREAD

**Issue by ankero** [froala/wysiwyg-editor#2101]
Hi,

We've run into an interesting issue with the froala editor. It seems that when you paste an url from Google Maps with @-signs, commas etc, the editor crashes with Chrome. I tried to figure out where the error happens, but with no particular luck. Its somewhere in the "clean()" function that causes the crash. 

And a bit more test details here:

* Url to test with, https://www.google.fi/maps/place/Kluuvi,+Helsinki/@60.1732541,24.9335843,15z/data=!3m1!4b1!4m5!3m4!1s0x46920bcdd0b20e9f:0x2600b5523c1977c1!8m2!3d60.1724919!4d24.9406385?hl=en (you can also just open Google Maps and create your own url)
* Updated to newest froala version 2.6.4 > still happens
* Chrome version 59.0.3071.115
* Tested with Chrome in your website > works
* Tested to type in address, not paste > works
* Mac
* You can also create the crash with any url with @, !, comma, + signs, but the maps url is a definite crasher

Our technical environment (simplified):

* Angular 1.8
* Tested with simple component (only initiates froala with default settings )

**Comment by stefanneculai** [comment]
@ankero please make sure the editor version was updated. You should be able to check that easily by writing `$.FE.VERSION` in the browser console. If the problem still persist, please try to reproduce this on a jsFiddle/plunker and open the issue directly on the [angular repo](https://github.com/froala/angular-froala). Thanks!

**Comment by ankero** [comment]
@stefanneculai ok so this is now resolved. To be honest I don't know fully what the issue is, but here's a description on what we found and how the issue was fixed:

Findings:
* This does not happen on every computer using Chrome (?)
* I wasn't able to reproduce with plunker - would indicate that this is related to other packages we use
* Seems to be related on the URL converting to a link (a -tag). If the link does not convert to a link, the error does not happen.
* Relates to special characters in long URLs
* Browser does not crash, it simply hangs in a long loop somewhere inside Froala


Fix:

* Start froalaEditor with linkConvertEmailAddress: false


I would not mark this as a bug in Froala, more like a "this might happen" -situation. The fix is easy and seems to mitigate the problem.
